### DISCUSSION TITLE
I created a new configuration file, how do I request it to be added?

### DISCUSSION THREAD

**Discussion by Bellavista** [make-all/tuya-local#1313] [Q&A] [answered]
Hi @make-all 
I created a new config file for honeyguaridan pet feeder s56
![image](https://github.com/make-all/tuya-local/assets/46230198/c6daf234-aaeb-4f97-87b0-b9402046b4d9)
i created a github repository whit some information
https://github.com/Bellavista/Tuya-Local-Honeygaridan-S56
What i need to do to add the config in tuya-local?
Best regards

**Comment by make-all** [comment, marked as answer]
The standard way to contribute your contributions is to clone this repository, and add your config into the custom_components/tuya_local/devices directory. Then you can open a pull request from your copy of the repository to this one.

Another way is to include the config along with the other information requested in a new device request under Issues above.

In both cases, it is useful for the review if you include the same log and iot portal information in your request as you would for requesting a new device without writing your own config.
